src/utilities/translation.js

```diff
@@ -26,6 +26,16 @@ const dict = {
     NEWS: {
       NEWS_TITLE: "Aktuelles",
       ITEMS: [
+        {
+          date: `16/09/2021`,
+          title: `Ab dem 30.09.2021 wird das Theaterstück "Spezies" der Gruppe "Oh!Darling", das ist wissenschaftlich beraten habe, in Zürch aufgeführt: https://www.kulturzueri.ch/kulturdatenbank-zurich/veranstaltungen/41451-3150-oh-darling/.`,

```

Comment:
@DrMarSeb Den link müsstest du an dieser Stelle raus nehmen und in der nächsten Zeile unter `link` einfügen. 
Sonst siehst das komisch aus: 

![image](https://user-images.githubusercontent.com/46668283/133920228-a267d69f-cd66-444f-bdbd-50b23fa063a3.png)